### ISSUE TITLE
Multi servers

### ISSUE THREAD

**Issue by Oleekae** [animetosho/Nyuu#139]
First of all, thank you for the great program you have developed. 

Im trying to edit the config file to add multi servers for more than one block account over connections >1Gbps, despite respecting the .json format it seems that nyuu doesn't like it as it doesn't upload anything, it only supports the single server format. 

The question is if it is possible to add more than 1 server and if not then if it will be implemented in the near future? 

Thank you very much and good holidays to all!

**Comment by animetosho** [comment]
If you're not already, multiple servers is only supported via config.js, not the JSON config file.

https://github.com/animetosho/Nyuu/blob/cde553076e7e087a3324cf3880cfeecff3a1be4c/config.js#L61

**Comment by Oleekae** [comment]
I'm just getting started in this whole puzzle, we'll see if I manage to make it all work. If not, I will comment here. 

Thank you very much.

**Comment by animetosho** [comment]
You can use config.js in place of the JSON file - in other words, just unify everything in config.js

**Comment by Oleekae** [comment]
Perfect, I will report results when I have the time to do so.

**Comment by Oleekae** [comment]
I've been struggling to do a nyuu configuration via .js instead of .json and I keep running into problems. So, for my part, Im using the windows compiled binary, is it possible to work with the config.js or do I need other dependencies like installing also on my Win10 system the Node.js to be fully compatible in its functions with the .js?  

I hope i made myself clear.

**Comment by animetosho** [comment]
It sounds like you figured it out, but there's nothing special required, you just use it like you would the config.json file (e.g. use it with the `-C` option).

> Is there a section where I can ask questions? As this is more for reporting bugs.

There's hardly enough posts here to warrant separating them into sections - posting here is fine.

**Comment by Oleekae** [comment]
Yes, after a few days of trial and error i was able to get the .js to work as it should. I don't work under commands, all coded under the configuration file and run it with a script. 

Perfect, so I'll close this thread and any other doubts, if I don't see them already posted here, I'll open them. 

Thanks.
